test/features/known_failures_Linux.json

```diff
@@ -1,46 +1,48 @@
 [
-  "c_glib-limit_container_length_binary_buffered-ip",
```

Comment:
In the future try not to reformat these files indentation; it makes it more difficult to understand what changed.

Reply:
It's auto-formatting in vscode maybe